File: JavaScript/programmers_031.js
function solution(price, money, count) {
    var answer = 0;
    for(let i = 1; i <= count; i++) {
        answer += price * i
    }
    return money - answer > 0 ? 0 : answer - money
}

// 등차수열
function solution(price, money, count) {
    const cost = price * count * (count + 1) / 2 - money;
    return cost > 0 ? cost : 0;
}

File: Python/Practice/programmers_031.py
def solution(x):
    sum = 0
    for n in str(x):
        sum += int(n)
    answer = True if x % sum == 0 else False
        
    return answer

# 간단하게 줄인 코드
def solution(x):
    return x % sum(int(n) for n in str(x)) == 0